CONTRIBUTING.adoc

```diff
@@ -1,13 +1,46 @@
-= Code formatting guidelines
+= Contributing to Spring Cloud GCP
+
+We graciously welcome contributions to Spring Cloud GCP.
```

Comment:
I don't know why, but "graciously" is giving me the eyerolls.

Reply:
Removed :)